de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -313,34 +221,39 @@ public void onAppCategorySelectionChanged(AppCategorySelectionChangedEvent event
 
     @Override
     public void onCommunitySelectionChanged(CommunitySelectionChangedEvent event) {
-        tileView.onCommunitySelectionChanged(event);
-        gridView.onCommunitySelectionChanged(event);
-
         Group selectedCommunity = event.getCommunitySelection();
-
         Preconditions.checkNotNull(selectedCommunity);
         if (!selectedCommunity.getId().equals(CommunitiesView.COMMUNITIES_ROOT)) {
             getCommunityApps(selectedCommunity);
         } else {
-            loadApps(Lists.newArrayList());
+            listView.setApps(null, false);
+            listView.setHeading(Joiner.on(" >> ").join(event.getPath()));
+            listView.setLoadingMask(true);
+
+            if (!selectedCommunity.getId().equals(CommunitiesView.COMMUNITIES_ROOT)) {
+                getCommunityApps(selectedCommunity);
+            } else {
+                listView.setApps(null, false);
+            }
```

Comment:
I don't think you need the above if/else?  The `if` condition on line 226 is the same as the `if` here on line 233, so this second condition will never be true.